src/acvp.c

```diff
@@ -100,10 +105,18 @@ ACVP_ALG_HANDLER alg_tbl[ACVP_ALG_MAX] = {
     {ACVP_SHA256,          &acvp_hash_kat_handler,  ACVP_ALG_SHA256},
     {ACVP_SHA384,          &acvp_hash_kat_handler,  ACVP_ALG_SHA384},
     {ACVP_SHA512,          &acvp_hash_kat_handler,  ACVP_ALG_SHA512},
-
     {ACVP_HASHDRBG,        &acvp_drbg_kat_handler,  ACVP_ALG_HASHDRBG},
     {ACVP_HMACDRBG,        &acvp_drbg_kat_handler,  ACVP_ALG_HMACDRBG},
     {ACVP_CTRDRBG,         &acvp_drbg_kat_handler,  ACVP_ALG_CTRDRBG},
+    {ACVP_HMAC_SHA1,       &acvp_hmac_kat_handler,  ACVP_ALG_HMAC_SHA1},
+    {ACVP_HMAC_SHA2_224,   &acvp_hmac_kat_handler,  ACVP_ALG_HMAC_SHA2_224},
+    {ACVP_HMAC_SHA2_256,   &acvp_hmac_kat_handler,  ACVP_ALG_HMAC_SHA2_256},
+    {ACVP_HMAC_SHA2_384,   &acvp_hmac_kat_handler,  ACVP_ALG_HMAC_SHA2_384},
+    {ACVP_HMAC_SHA2_512,   &acvp_hmac_kat_handler,  ACVP_ALG_HMAC_SHA2_512},
+    {ACVP_HMAC_SHA3_224,   &acvp_hmac_kat_handler,  ACVP_ALG_HMAC_SHA3_224},
```

Comment:
We discussed including SHA3 entries, but I may not have mentioned we may want to make sure they don't blow up in case someone tries to use them, ie if some ACVP user has SHA3 in their module and tries to register we want to return some error that says not supported.